### ISSUE TITLE
Retro Consoles (<480i) do not output in FIFO

### ISSUE THREAD

**Issue by bmosley** [tolga9009/elgato-gchd#35]
Unable (so far in testing) to get any retro consoles (SNES, Genesis, NES) to work with the driver. Working fine using elgato software.

**Comment by NerdyShawn** [comment]
I made a pacp a while back of the card initializing a 240p signal with the windows drivers on a vm, the captured was on ubuntu 14.04 box with I believe... a Windows 8 guest if I recall correctly. https://www.dropbox.com/s/suj1vynst8zpn7w/240p_starwars-plugnplay.pcapng?dl=0
I tried capturing the initialization. This was done with a star wars plug-n-play composite (the sweet millennium falcon little one https://www.wired.com/images_blogs/gadgetlab/retro-gaming-millennium-falcon-joystick.png)

**Comment by knoebber** [comment]
Any update on this? I'm trying to get a n64 to capture.

**Comment by williamsjokvist** [comment]
I think it's because this is based on an older version of the driver where there was no 240p support. 

I hope someone patches it, it's the only reason I'm still having to use Winblows :(

**Comment by Magicrafter13** [comment]
Only got this software to stream some NES gameplay only to find out I can't...
So yeah, hope this is taken care of?

**Comment by DusteDdk** [comment]
@Magicrafter13 yeah, this open source software that someone made on their own, and decided to give away for free? Yeah, maybe you go take your entitlement somewhere else, or, even better, maybe you fix it yourself and submit a pull request ?
